src/object.rs

```diff
@@ -379,11 +380,14 @@ impl Object {
     .into()
   }
 
-  /// The `Object` specific equivalent of `Data::get_hash()`.
-  /// This function is kept around for testing purposes only.
+  /// Returns the V8 hash value for this value. The current implementation
+  /// uses a hidden property to store the identity hash.
+  ///
+  /// The return value will never be 0. Also, it is not guaranteed to be
+  /// unique.
   #[doc(hidden)]
```

Comment:
Was removal of this attribute omitted on purpose?

Reply:
No, it was omitted by accident, thanks for pointing that out.